lib/pure/encodings.nim

```diff
@@ -421,12 +451,17 @@ when defined(windows):
            else: convertFromWideString(codePageTo, wideString)
 
   proc convert*(c: EncodingConverter, s: string): string =
-    ## converts `s` to `destEncoding` that was given to the converter `c`. It
+    ## Converts `s` to `destEncoding` that was given to the converter `c`. It
     ## assumed that `s` is in `srcEncoding`.
-    ## utf-16BE, utf-32 conversions not supported on windows
+    ##
+    ## .. warning:: utf-16BE, utf-32 conversions are not supported on windows.
     result = convertWin(c.src, c.dest, s)
 else:
   proc convert*(c: EncodingConverter, s: string): string =
+    ## Converts `s` to `destEncoding` that was given to the converter `c`. It
+    ## assumed that `s` is in `srcEncoding`.
+    ##
+    ## .. warning:: utf-16BE, utf-32 conversions are not supported on windows.
```

Comment:
![image](https://user-images.githubusercontent.com/43030857/124873624-4dcc0880-dff9-11eb-9b8b-7d7364f58230.png)

It didn't generate docs before. So I add a comment here.